apps/ledger-live-mobile/package.json

```diff
@@ -146,7 +146,7 @@
     "expo-keep-awake": "~12.8.2",
     "expo-modules-autolinking": "^1.10.2",
     "expo-modules-core": "^1.11.8",
-    "expo-random": "^13.6.0",
+    "expo-crypto": "^12.8.1",
```

Comment:
Why not use the latest version (`13.0.2`) of `expo-crypto` ?

Reply:
good point. 
In fact, I just take the same PR as zondax did. https://github.com/LedgerHQ/ledger-live/pull/7459/files
I will try with your suggestion, thanks

Reply:
In fact, 
in the new version of expo-crypto
the https://github.com/expo/expo/blob/main/packages/expo-crypto/android/build.gradle
is not compatible with our current android SDK version and it causes an error in CI.

it seems that the "useDefaultAndroidSdkVersions" needs a higher version of android SDK version.

Reply:
Okay too bad maybe we'll need to update that at some point with the next RN update